### PR TITLE
Create installation.md

### PR DESCRIPTION
**Pull request by rootli**
Erster Entwurf für die Installationsanleitung

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  bottom: false
+---
+
+# How to install <span class="my-karakun">Open</span>WebStart
+
+<span class="my-karakun">Open</span>WebStart alpha comes with a user-friendly installer for Windows. Download the "stable" build to try out the <span class="text-highlight">Open<span>WebStart</span></span> alpha release now!
+
+## How to Install <span class="my-karakun">Open</span>WebStart on Microsoft Windows
+
+1. Go to https://openwebstart.com/download and download `OpenWebStart_windows-x64_0_1_0-alpha.zip`.
```
Comment (sclassen) [inline installation.md]: can you replace the URL with the following:
`[download](/download)`
This will create a link to the download page

Comment (sclassen) [inline installation.md]: If you replace the specific version `OpenWebStart_windows-x64_0_1_0-alpha.zip` with a more generic `latest version` the guide could stay unchanged when we push a new version of open web start

Comment (netopyr) [reply installation.md]: I fixed this already in the release branch.

File: installation.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+permalink: /installation/
+header:
+  text: How to install <span class="my-karakun">Open</span>WebStart
+nav:
+  bottom: false
+---
+
+# How to install <span class="my-karakun">Open</span>WebStart
+
+<span class="my-karakun">Open</span>WebStart alpha comes with a user-friendly installer for Windows. Download the "stable" build to try out the <span class="text-highlight">Open<span>WebStart</span></span> alpha release now!
```
[the remark this answers is not in the document]
Comment (netopyr) [reply installation.md]: The average user will probably never see the GitHub page, but only our download page, therefore we should not rely on GitHub-features.
We should warn the users everywhere possible that this is just an alpha version. 🙂

[the remark this answers is not in the document]
Comment (AndreasEhret) [reply installation.md]: ok, then it is fine to stay here with this alpha hint for now

**Comment by sclassen** [comment]
Closing this PR as @netopyr has already merged the changes in the branch "release_0.1"

File: installation.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## How to Install <span class="my-karakun">Open</span>WebStart on Microsoft Windows
+
+1. Go to https://openwebstart.com/download and download `OpenWebStart_windows-x64_0_1_0-alpha.zip`.
+1. Run the installer.
+1. Choose a language and click **OK** to open the <span class="my-karakun">Open</span>WebStart Setup wizard. 
+1. Click **Next** to start the <span class="my-karakun">Open</span>WebStart installation.
+1. Browse to the directory where to install <span class="my-karakun">Open</span>WebStart, and click **Next**. 
+   <br />Windows default: `C:\Program Files\OpenWebStart`
+1. Enable the checkbox to associate the .JNLP suffix with <span class="my-karakun">Open</span>WebStart, and click **Next**.
```
[the remark this answers is not in the document]
Comment (AndreasEhret) [reply installation.md]: will be improved in documentation for 0.2.0

File: installation.md (hunk 1 above)
Comment (AndreasEhret) [reply installation.md]: will be improved in documentation for 0.2.0

**Pull request closed without merging**